### DISCUSSION TITLE
How to Upload Products in Batches of 20 instead of 100?

### DISCUSSION THREAD

**Discussion by r00tR** [Second-Hand-Friends/kleinanzeigen-bot#347] [Q&A]
I’m using your kleinanzeigen-bot project and I noticed that when I trigger the bot, it uploads products in bulk (e.g., 100 items at once). However, I would prefer to upload the products in smaller batches, such as 20 items per batch.

Could you please guide me on how to modify the bot to upload the products in batches of 20 instead of 100?

Any help or direction would be greatly appreciated!

Thank you for your work on this project!

Best regards,

**Comment by sebthom** [comment]
There is no code in place to even limit for 100. The bot loops over all found ads and the ads that are outdated or new get (re-)created
